browser/help/topics/ancestry.md

```diff
@@ -1,58 +1,43 @@
 ---
 id: ancestry
-title: 'Genetic Ancestry in gnomAD'
+title: 'Genetic Ancestry in AGDD'
 ---
 
-See our [blog post](https://gnomad.broadinstitute.org/news/2023-11-genetic-ancestry) on genetic ancestry for more details on our process and additional commentary on the benefits and limitations of using genetically-derived ancestry.
+### Genetic ancestry Groups
+See our [blog post](https://ourdna-dev.popgen.rocks/news/2025-08-genetic-ancestry) on genetic ancestry for more details on our process and additional commentary on the benefits and limitations of using genetically-derived ancestry.
```

Comment:
Hey @joshuamschmidt, just replace (https://ourdna-dev.popgen.rocks/news/2025-08-genetic-ancestry)
with
(/news/2025-08-genetic-ancestry)
No need to use the root domain, just relative link to the blog.